src/utils/constants.ts

```diff
@@ -13,7 +13,7 @@ export const BEAMER_ID = IS_PRODUCTION ? process.env.REACT_APP_BEAMER_ID : 'ehlR
 export const SENTRY_DSN = process.env.REACT_APP_SENTRY_DSN || ''
 export const PORTIS_ID = process.env.REACT_APP_PORTIS_ID ?? '852b763d-f28b-4463-80cb-846d7ec5806b'
 export const FORTMATIC_KEY = process.env.REACT_APP_FORTMATIC_KEY ?? 'pk_test_CAD437AA29BE0A40'
-export const INFURA_TOKEN = process.env.REACT_APP_INFURA_TOKEN || ''
+export const RPC_ENDPOINT = process.env.REACT_APP_PRC_NODE || ''
 export const SAFE_APPS_RPC_TOKEN = process.env.REACT_APP_SAFE_APPS_RPC_INFURA_TOKEN || ''
```

Comment:
please accommodate for this INFURA_TOKEN as well

Reply:
I have removed the other instance of INFURA_TOKEN as well. Please take a look at it.